torchvision/models/detection/rpn.py

```diff
@@ -302,9 +151,6 @@ def __init__(self,
         self.head = head
         self.box_coder = det_utils.BoxCoder(weights=(1.0, 1.0, 1.0, 1.0))
 
-        # used during training
-        self.box_similarity = box_ops.box_iou
```

Comment:
Do we need this change? Can we keep as it was before?

Reply:
To be honest I don't remember. I will look into it. I vaguely remember some problem with type hints, which is a very silly reason, but didn't want to look into it at the time. I'm not familiar with the type hinting that is used in pytorch, how is it used throughout the project? Is there some documentation you can refer me to?

Reply:
the type hints are only needed for torchscript, and in the current state of things it should work as expected I believe.